static/javascript/tba_js/tba.js

```diff
@@ -153,3 +153,13 @@ $(document).ready(function(){
     $("#robot-image").html('<div class="thumbnail carousel team-media-carousel"><a href="' + url + '" target="_blank"><img src="' + url + '" alt="April Fools!"></a></div>');
   }
 });
+
+// Init Firebase
+var config = {
```

Comment:
I mean none of this is private because it's all client side javascript code. The validation comes from the pair of 1) GCM key on the server and 2) FCM token.